src/workers/personalizedDigestEmail.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  addNotificationUtm,
+  baseNotificationEmailData,
+  feedToFilters,
+  fixedIdsFeedBuilder,
+  getDiscussionLink,
+  pickImageUrl,
+  sendEmail,
+} from '../common';
+import { ArticlePost, Post, Source, User } from '../entity';
+import { UserPersonalizedDigest } from '../entity/UserPersonalizedDigest';
+import { messageToJson, Worker } from './worker';
+import { DayOfWeek } from '../types';
+import { MailDataRequired } from '@sendgrid/mail';
+import { format, nextDay, previousDay } from 'date-fns';
+import { zonedTimeToUtc } from 'date-fns-tz';
+import { personalizedDigestFeedClient } from '../integrations/feed/generators';
+
+interface Data {
+  personalizedDigest: UserPersonalizedDigest;
+}
+
+type TemplatePostData = Pick<
+  ArticlePost,
+  'id' | 'title' | 'image' | 'createdAt'
+> & {
+  sourceName: Source['name'];
+  sourceImage: Source['image'];
+};
+
+const personalizedDigestPostsCount = 5;
+
+const emailTemplateId = 'd-328d1104d2e04fa1ab91e410e02751cb';
+
+const personalizedDigestDateFormat = 'yyyy-MM-dd HH:mm:ss';
+
+const getEmailSendDate = ({
+  personalizedDigest,
+}: {
+  personalizedDigest: UserPersonalizedDigest;
+}): Date => {
```

Comment:
I think we can use the same interface or at least base from it.

```suggestion
}: Data): Date => {
```